examples/traefik-new/docker-compose-redis.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+
+services:
+  redis:
+    image: redis:alpine
+    restart: unless-stopped
+    networks:
+      - traefik-public
+    deploy:
+      resources:
+        limits:
+          memory: 8G # Adjust as needed if you'd like to use redis as a cache for blob descriptors
+        reservations:
+          memory: 4G
```

Comment:
Holy Flying Spaghetti Monster!

4GB RESERVATION???

[According to Redis documentation](https://redis.io/docs/latest/develop/get-started/faq/), an empty Redis instance uses about 3MB, and a 1 million key-hash pairs (5 fields) instance uses around 160MB.

Why are you RESERVING 4GB???